pkg/secrethub/org_member.go

```diff
@@ -112,3 +115,41 @@ func (s orgMemberService) Update(org string, username string, role string) (*api
 
 	return s.client.httpClient.UpdateOrgMember(org, username, in)
 }
+
+// Iterator returns an iterator that lists all members of a given organization.
+func (s orgMemberService) Iterator(org string, _ *OrgMemberIteratorParams) OrgMemberIterator {
+	data, err := s.List(org)
+	return &orgMemberIterator{
+		index: 0,
+		data:  data,
+		err:   err,
+	}
+}
+
+// OrgMemberIteratorParams defines parameters used when listing OrgMembers.
```

Comment:
When writing godoc, we usually try to use natural language as much as possible, rather than naming the specific type returned. In this case, that would mean saying `when listing members of the organization` rather than `when listing OrgMembers`

Reply:
Thanks for the tip! I'll rephrase the comment.